proposals/0251-font-styles.md

```diff
@@ -52,6 +52,9 @@ With this implementation, technically the HTML tags take up part of the 500 char
   .
   <param name="fieldText" type="String" maxlength="1000" mandatory="true">
     <description>The text itself.</description>
+    <history>
+      <param name="fieldText" type="String" maxlength="500" mandatory="true">
```

Comment:
This isn't correct, please look at other history tags. It needs version information.